tantar/src/main/java/org/opencadc/tantar/policy/PolicyMetadata.java

```diff
@@ -0,0 +1,154 @@
+ *  any later version.                   toute version ultérieure.
+ *
+ *  OpenCADC is distributed in the       OpenCADC est distribué
+ *  hope that it will be useful,         dans l’espoir qu’il vous
+ *  but WITHOUT ANY WARRANTY;            sera utile, mais SANS AUCUNE
+ *  without even the implied             GARANTIE : sans même la garantie
+ *  warranty of MERCHANTABILITY          implicite de COMMERCIALISABILITÉ
+ *  or FITNESS FOR A PARTICULAR          ni d’ADÉQUATION À UN OBJECTIF
+ *  PURPOSE.  See the GNU Affero         PARTICULIER. Consultez la Licence
+ *  General Public License for           Générale Publique GNU Affero
+ *  more details.                        pour plus de détails.
+ *
+ *  You should have received             Vous devriez avoir reçu une
+ *  a copy of the GNU Affero             copie de la Licence Générale
+ *  General Public License along         Publique GNU Affero avec
+ *  with OpenCADC.  If not, see          OpenCADC ; si ce n’est
+ *  <http://www.gnu.org/licenses/>.      pas le cas, consultez :
+ *                                       <http://www.gnu.org/licenses/>.
+ *
+ *
+ ************************************************************************
+ */
+
+package org.opencadc.tantar.policy;
+
+import java.net.URI;
+import java.util.Objects;
+
+import org.apache.log4j.Logger;
+import org.opencadc.inventory.Artifact;
+import org.opencadc.inventory.storage.StorageMetadata;
+
+
+/**
+ * Simple encompassing class to house the metadata to be compared when enforcing a policy for an existing Storage
+ * Location.
+ */
+public class PolicyMetadata {
+
+    private static final Logger LOGGER = Logger.getLogger(PolicyMetadata.class);
+
+    private final URI contentChecksum;
+    private final Long contentLength;
+    public final String contentType;
+    public final String contentEncoding;
+
+    PolicyMetadata(URI contentChecksum, Long contentLength, String contentType,
+                   String contentEncoding) {
+        this.contentChecksum = contentChecksum;
+        this.contentLength = contentLength;
+        this.contentType = contentType;
+        this.contentEncoding = contentEncoding;
+    }
+
+
+    /**
+     * Create a new Policy Metadata by extracting valid fields from an Artifact.
+     *
+     * @param artifact The Artifact to use.
+     */
+    PolicyMetadata(final Artifact artifact) {
+        this(artifact.getContentChecksum(), artifact.getContentLength(), artifact.contentType,
+             artifact.contentEncoding);
+    }
+
+    /**
+     * Create a new Policy Metadata by extracting valid fields from a StorageMetadata.
+     *
+     * @param storageMetadata The StorageMetadata to use.
+     */
+    PolicyMetadata(final StorageMetadata storageMetadata) {
+        this(storageMetadata.getContentChecksum(), storageMetadata.getContentLength(),
+             storageMetadata.contentType, storageMetadata.contentEncoding);
+    }
+
```

Comment:
I'm not sure this equals method is all that useful with the different way that discrepancies are interpretted... maybe the whole class is now obsolete?